src/avfoundation.cs

```diff
@@ -661,6 +850,9 @@ interface AVAudioConnectionPoint
 		[Export ("bus")]
 		nuint Bus { get; }
 	}
+
+	[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+	delegate AVAudioEngineManualRenderingStatus AVAudioEngineManualRenderingBlock (/* AVAudioFrameCount = uint */ uint arg0, /* AudioBufferList */ IntPtr arg1, [NullAllowed] /* OSStatus */ int arg2);
```

Comment:
Don't use `arg0`...
The point of using custom delegate is that they are self documenting because of the parameter names
iirc sharpie won't give them - but the headers (or the type) should help